diesel/src/expression/cast.rs

```diff
@@ -96,10 +96,11 @@ where
 }
 
 macro_rules! type_name {
-    ($($backend: ty: $backend_feature: literal { $($type: ident => $val: literal,)+ })*) => {
+    ($($backend: ty: $backend_feature: literal { $($($feature: literal;)? $type: ident => $val: literal,)+ })*) => {
```

Comment:
It looks like `?` would feel more intuitive to read than `;` if possible.

Reply:
Closing because will be removed as per other comment. (I checked and indeed Uuid exists without the feature.)